state-synchronizer/src/synchronizer.rs

```diff
@@ -159,6 +159,10 @@ impl StateSyncClient {
                 ))
                 .await?;
 
+            // @REVIEW: Is one second reasonable? What happens if there is an error?
+            // in block_store this seems to cause a panic. Imagine that there was swapping / slow disk
+            // seems like this would cause a panic.
+            // Does this actually need to be synchronous?
```

Comment:
first time seeing this.
we probably should relax the timeout, reconfig publish depends on this (it probably doesn't matter for correctness although may make debug more difficult).